go-controller/pkg/metrics/master.go

```diff
@@ -246,37 +259,24 @@ func RegisterMasterMetrics(nbClient, sbClient goovn.Client) {
 		prometheus.MustRegister(metricV6AllocatedHostSubnetCount)
 		prometheus.MustRegister(metricEgressIPCount)
 		prometheus.MustRegister(metricEgressFirewallRuleCount)
+		prometheus.MustRegister(metricIPsecEnabled)
 		registerWorkqueueMetrics(MetricOvnkubeNamespace, MetricOvnkubeSubsystemMaster)
 	})
 }
 
-// StartE2ETimeStampMetricUpdater adds a goroutine that updates a "timestamp" value in the
-// nbdb every 30 seconds. This is so we can determine freshness of the database
-func StartE2ETimeStampMetricUpdater(stopChan <-chan struct{}, ovnNBClient goovn.Client) {
-	startE2ETimeStampUpdaterOnce.Do(func() {
+// StartMasterMetricUpdater adds a goroutine that updates a "timestamp" value in the
```

Comment:
Nit: Another question...why do we need to do this every 30 seconds for Prometheus metrics? Shouldn't it just report the state of the db instead of writing a timestamp? (sorry this is my lack of understanding with Prometheus)

Reply:
I didn't write this feature - just updating to use libovsdb while I am here. 
Writing the timestamp is a way of verifying if the DB is functional is my understanding before writing the timestamp metric.
How could you report the state of the DB?

Reply:
actually @squeed had this https://github.com/ovn-org/ovn-kubernetes/pull/2414